e2e_tests/helpers.go

```diff
@@ -12,3 +14,11 @@ func RequireCurrentBranchName(t *testing.T, repo *git.Repo, name string) {
 	require.NoError(t, err, "failed to determine current branch name")
 	require.Equal(t, name, currentBranch, "expected current branch to be %q, got %q", name, currentBranch)
 }
+
+func GetStoredParentBranchState(t *testing.T, repo *git.Repo, name string) meta.BranchState {
```

Comment:
Not for this PR, but we should add a command to get this information that way we don't have to do this.

Reply:
Sure